syntax = "proto3";

package qoin;

service Greeter {
  rpc SayHello(HelloRequest) returns (HelloReply) {}
  rpc SayHelloAgain(HelloRequest) returns (HelloReply) {}
  rpc HelloStream(HelloRequest) returns (stream HelloReply) {}
}

message HelloRequest { string name = 1; }
message HelloReply { string message = 1; }
